From nobody Mon Sep 17 00:00:00 2001
From: tomek@koprowski.org <tomek@koprowski.org>
Date: Sun Feb 19 18:03:24 2006 +0100
Subject: [PATCH 25/49] PCI: SMBus unhide on HP Compaq nx6110
Patch-mainline: 2.6.17

I attach a trivial patch for 2.6.15.4 that unhides SMBus controller
on an HP Compaq nx6110 notebook.

Signed-off-by: Tomasz Koprowski <tomek@koprowski.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---

 drivers/pci/quirks.c |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

3c0a654e390d00fef9d8faed758f5e1e8078adb5
diff --git a/drivers/pci/quirks.c b/drivers/pci/quirks.c
index 7ec9fc9..af4bc8e 100644
--- a/drivers/pci/quirks.c
+++ b/drivers/pci/quirks.c
@@ -934,6 +934,12 @@ static void __init asus_hides_smbus_host
 			case 0x12bd: /* HP D530 */
 				asus_hides_smbus = 1;
 			}
+		if (dev->device == PCI_DEVICE_ID_INTEL_82915GM_HB) {
+			switch (dev->subsystem_device) {
+			case 0x099c: /* HP Compaq nx6110 */
+				asus_hides_smbus = 1;
+			}
+		}
 	} else if (unlikely(dev->subsystem_vendor == PCI_VENDOR_ID_TOSHIBA)) {
 		if (dev->device == PCI_DEVICE_ID_INTEL_82855GM_HB)
 			switch(dev->subsystem_device) {
-- 
1.2.4

